md_rahaman/python/django/surprise_me/apps/surprise_me_app/views.py

```diff
@@ -0,0 +1,33 @@
+from django.shortcuts import render,redirect
+import datetime
+import random
+
+# Create your views here.
+
+VALUES = ['one','Two','Three','Four','Five','Six']
```

Comment:
Good to understand what the global variable is. I know this assignment encouraged the use of a global variable, but I'd like to reiterate that in general global variables are a bad idea in production environments.